modules/administration-guide/pages/configuring-the-administrative-user.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring the administrative user
+:keywords: administration-guide, admin
+:page-aliases:
+
+[id="configuring-the-administrative-user_{context}"]
+= Configuring the administrative user
+
+The user with administrative privileges has all {prod-short} system permissions.
+The default administrative user is `admin`.
```

Comment:
Is the privileged mode enabled or disabled by default?

Reply:
On a Che 7.48 instance on crc, OpenShift contains only 2 users: `kubeadmin` and `developer`. The default `admin` user doesn't exist.

Reply:
I am creating the `developer` user again after each test.